tensorpipe/proto/channel/cuda_ipc.proto

```diff
@@ -19,6 +19,7 @@ message Reply {
   bytes stop_ev_handle = 1;
 }
 
-message Ack {
-  uint64 sequence_number = 1;
+message Packet {
+  // Empty reply means Ack.
```

Comment:
I have a slight preference to have an explicit `Ack` message type, even if empty. It's more explicit and thus avoids the confusion where forgetting to fill in a packet is interpreted as the packet being an ack. Having an Ack type and writing a switch on the `oneof` and raising an error in the `default` block would be more robust.